.github/scripts/setup_environment.sh

```diff
@@ -124,8 +124,8 @@ function setup_ubuntu_install_dependencies() {
 					log_warning "Installed linux-headers-generic."
 				fi
 			else
-				log_warning "Installation aborted by user.."
-				exit 0
+				log_warning "ICE cannot be installed in when CICD_BUILD is set, quitting..."
+				exit 1
```

Comment:
This message doesn't make sense there,

It will be much clearer when you put it somewhere above where ret 0 is set

This part is accessed when the user decides not to continue with the general kernel header installation, 
if we want to be more accurate, then 
``` suggestion
				log_warning "Cannot install kernel headers for the ICE driver, aborting installation..."
				exit 1
```

Reply:
revoke